program/merge-two-strings/MergeTwoStrings.js

```diff
@@ -0,0 +1,5 @@
+function mergeStrings(str1, str2) {
+  return str1 + str2;
+}
+
+mergeStrings();
```

Comment:
can you pass the string which is given in the issue as an argument while invoking the function 
`console.log(mergeStrings('ABC', 'DEF'));`